phpBB/includes/functions_privmsgs.php

```diff
@@ -2109,6 +2109,7 @@ function message_history($msg_id, $user_id, $message_row, $folder, $in_post_mode
 			'MSG_ID'			=> $row['msg_id'],
 			'MESSAGE_TIME'		=> $row['message_time'],
 			'USER_ID'			=> $row['user_id'],
+			'S_NOT_SELF'	=> ($row['user_id'] != $user->data['user_id']) ? true : false,
```

Comment:
I would advise against negated variable names. If at some point we need to check if a message is from the current user, we'd have to use `not S_NOT_SELF` which can get confusing.

Instead, I would suggest using:
```
'S_IS_SELF'	=> ($row['user_id'] == $user->data['user_id']) ? true : false,
```
and in the template do `if not S_IS_SELF`.

Reply:
makes sense!